src/lib.rs

```diff
@@ -39,11 +42,15 @@ pub use crate::redismodule::*;
 static ALLOC: crate::alloc::RedisAlloc = crate::alloc::RedisAlloc;
 
 /// `LogLevel` is a level of logging to be specified with a Redis log directive.
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, AsRefStr)]
 pub enum LogLevel {
+    #[strum(to_string = "debug")]
     Debug,
+    #[strum(to_string = "notice")]
     Notice,
+    #[strum(to_string = "verbose")]
     Verbose,
+    #[strum(to_string = "warning")]
```

Comment:
Looks like you could replace all those `#[strum(to_string = "debug")]` lines with a single `#[strum(serialize_all = "snake_case")]` on the `enum`.

Reply:
Good call.